buildman/builder.py

```diff
@@ -89,7 +87,7 @@ def run_build_manager():
         build_manager_config[1],
         instance_keys,
     )
-    server.run("0.0.0.0", controller_port, ssl=ssl_context)
```

Comment:
Why is this no longer required?

Reply:
Wasn't used. Probably from the old code. Instead, the ca is specified as a config option, or set to use the `ssl.cert` otherwise.